From mcarlson@broadcom.com  Thu Jun 23 22:01:22 2011
From: Matt Carlson <mcarlson@broadcom.com>
Date: Wed, 22 Jun 2011 18:56:59 -0700
Subject: [PATCH 130/194] tg3: use <linux/io.h> and <linux/uaccess.h> instead
 <asm/io.h> and <asm/uaccess.h>
Git-commit: 27fd9de8eb9bf48e4a09c8f02d13f56e9a074a1e
Patch-mainline: v2.6.39-rc1
References: bnc#697783, FATE#311457

It is proper style to include linux/foo.h instead asm/foo.h if both exist

Signed-off-by: Javier Martinez Canillas <martinez.javier@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/net/tg3.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/tg3.c b/drivers/net/tg3.c
index b84e1df..a568eb6 100644
--- a/drivers/net/tg3.c
+++ b/drivers/net/tg3.c
@@ -47,9 +47,9 @@
 #include <net/ip.h>
 
 #include <asm/system.h>
-#include <asm/io.h>
+#include <linux/io.h>
 #include <asm/byteorder.h>
-#include <asm/uaccess.h>
+#include <linux/uaccess.h>
 
 #ifdef CONFIG_SPARC
 #include <asm/idprom.h>
-- 
1.7.3.4

